packages/agents/lighthouse/src/tasks/prover/adapters/relayer/gelato.ts

```diff
@@ -1,8 +1,8 @@
 import { createLoggingContext } from "@connext/nxtp-utils";
-import { AxiosError } from "axios";
+import { GelatoRelaySDK } from "@gelatonetwork/relay-sdk";
```

Comment:
I think it would be much better to refactor this into a separate adapter package, since it's used across multiple agents and it can be generalized.

Reply:
Done.
Refactored this into relayer in utils, where all generalized gelato helpers are.